src/services/item/plugins/importExport/schema.ts

```diff
@@ -1,24 +1,41 @@
 import { Type } from '@sinclair/typebox';
+import { StatusCodes } from 'http-status-codes';
 
 import { FastifySchema } from 'fastify';
 
 import { customType } from '../../../../plugins/typebox';
+import { errorSchemaRef } from '../../../../schemas/global';
 
 export const zipImport = {
-  querystring: Type.Partial(
-    Type.Object(
-      {
-        parentId: customType.UUID(),
-      },
-      {
-        additionalProperties: false,
-      },
+  operationId: 'importZip',
+  tags: ['item', 'import'],
+  summary: 'Import ZIP content',
+  description: 'Import content of a ZIP, creating corresponding structure and items.',
+
+  querystring: customType.StrictObject({
+    parentId: Type.Optional(
```

Comment:
Interesting that the name of this parameter is not consistent with the file upload one.